src/client/src/shell/notification/Styled.tsx

```diff
@@ -0,0 +1,50 @@
+import { Flex } from 'rt-components'
+import { styled } from 'rt-util'
+
+export const Notification = styled('div')`
+  height: 100%;
+  background-color: ${({ theme: { background } }) => background.backgroundSecondary};
```

Comment:
Are we not confident about using the new theme structure?

Reply:
It's been merged and ready.

Reply:
Most of these commits were before I considered the new structure stable. I will have another task to migrate in.